src/card_definitions/amex.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
can you combine all the jsons into one card_definitions/card_mappings files?

Reply:
great idea! done!